frontend/app/routes/_gcweb-app.update-phone-number-success.tsx

```diff
@@ -0,0 +1,36 @@
+import type { LoaderFunctionArgs} from "@remix-run/node";
+import { json } from "@remix-run/node";
+import { useLoaderData, useNavigate } from "@remix-run/react";
+import { PhoneNumber } from "~/components/phone-number";
+import { getUserService } from "~/services/user-service.server";
+import { getEnv } from "~/utils/env.server";
+
+export async function loader({ request }: LoaderFunctionArgs) {
+    const env = getEnv();
+    const { getUserId, getUserInfo } = getUserService({ env });
+    const userId = await getUserId();
+    return json({
+      userInfo: await getUserInfo(userId),
+    });
+  }
+
+export default function UpdatePhoneNumberConfirm() {
```

Comment:
**Nitpick:** Should this component be named `UpdatePhoneNumberSuccess`?

Reply:
Will fix typo in next PR